engine/src/main/java/org/camunda/bpm/engine/impl/cmd/GetTelemetryDataCmd.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.cmd;
+
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.camunda.bpm.engine.impl.interceptor.Command;
+import org.camunda.bpm.engine.impl.interceptor.CommandContext;
+import org.camunda.bpm.engine.impl.telemetry.dto.TelemetryDataImpl;
+
+public class GetTelemetryDataCmd implements Command<TelemetryDataImpl> {
+
+  ProcessEngineConfigurationImpl configuration;
+
+  @Override
+  public TelemetryDataImpl execute(CommandContext commandContext) {
+    configuration = commandContext.getProcessEngineConfiguration();
+
+    TelemetryDataImpl telemetryData = configuration.getTelemetryReporter().getTelemetrySendingTask().updateAndSendData(false);
```

Comment:
It's possible that the reporter is null (see `ProcessEngineConfigurationImpl#initTelemetry`), in which case we would see a `NullPointerException` here. Maybe we can throw a proper exception in this case. What do you think?

Reply:
Good idea! I suggest something like `Error while retrieving telemetry data. Telemetry registry was not initialized.`.